src/app/users/login.ts

```diff
@@ -60,7 +61,10 @@ export const login = async ({
   if (user instanceof CouldNotFindUserFromPhoneError) {
     subLogger.info({ phone }, "new user signup")
     const userRaw: NewUserInfo = { phone }
-    user = await createUserForPhoneSchema(userRaw)
+    user = await createUserForPhoneSchema({
+      newUserInfo: userRaw,
+      config: getAccountsConfig(),
```

Comment:
I would rename the function more into something like `getDefaultAccountsConfig`.

the idea is that account will be able to evolve over time. ie: as a enterprise user, I may want to add new wallets to my account.